kiwi/xml_description.py

```diff
@@ -52,12 +53,20 @@ class XMLDescription(object):
     * :attr:`derived_from`
         path to base XML description file
 
+    * :attr:`xml_content`
+        XML description data as content string
     """
-    def __init__(self, description=None, derived_from=None, content=None):
+    def __init__(self, description=None, derived_from=None, xml_content=None):
```

Comment:
Well, if we are renaming `content` to `xml_content` (are we expect something like `json_content` or `yaml_content` too?) then the `description` is actually just a path to the file. Maybe it can be `path` now?

I would keep `content` instead.